src/containers/WelcomePage/FrontpageSubjects.jsx

```diff
@@ -163,10 +164,20 @@ class FrontpageSubjects extends Component {
       ? frontpageCategories
       : [...frontpageCategories, getAllImportSubjectsCategory(subjects)];
 
+    const allSubjectsWithFIxedEndSlash = allSubjects.map(category => ({
+      ...category,
+      subjects: category.subjects
+        ? category.subjects.map(subject => ({
+            ...subject,
+            url:
+              subject && subject.url ? fixEndSlash(subject.url) : subject.url,
+          }))
+        : [],
+    }));
```

Comment:
Mye jobb for å endre en liten ting, hva med å gjøre dette i FrontpageSubjectSection i stedet? Da kan man bare kjøre alle url-linker gjennom fixEndSlash